nest/src/recorders/interfaces/base.recorder.ts

```diff
@@ -0,0 +1,3 @@
+export interface BaseRecorder {
+    id: number;
```

Comment:
Assuming this is eqivelant to a PS2AlertsInstanceInterface, missing stuff.

Reply:
Yes, I think so.